Sources/FluentKit/Migration/Migrations.swift

```diff
@@ -14,4 +14,12 @@ public final class Migrations {
     public func add(_ migration: Migration, to id: DatabaseID? = nil) {
         self.storage.append(.init(id: id, migration: migration))
     }
+
+    public func add(_ migrations: Migration..., to id: DatabaseID? = nil) {
+        self.add(migrations, to: id)
+    }
+
+    public func add(_ migrations: [Migration], to id: DatabaseID? = nil) {
```

Comment:
If we make this call `add(_ migration:to:)` we could make this `@inlinable` too?

Reply:
Calling `.append(contentsOf: .map { ... })` should be sufficiently more efficient than `.forEach { .append($0) }` to warrant not chaining here.

Reply:
Ah, yeah that makes sense 👍